argilla-server/src/argilla_server/validators/records.py

```diff
@@ -25,6 +28,19 @@
 from argilla_server.errors.future.base_errors import UnprocessableEntityError
 from argilla_server.models import Dataset, Record
 
+IMAGE_FIELD_WEB_URL_MAX_LENGTH = 2038
+IMAGE_FIELD_DATA_URL_MAX_LENGTH = 5_000_000
```

Comment:
I think we also added this validation to the python client, can your remove it from there?